robolectric-annotations/src/main/java/org/robolectric/annotation/Config.java

```diff
@@ -102,6 +105,20 @@
   String qualifiers() default DEFAULT_QUALIFIERS;
 
   /**
+   * Android build type to use for resource resolution, such as "debug".
+   *
+   * @return Build type used for resource resolution.
+   */
+  String buildType() default DEFAULT_BUILD_TYPE;
+
+  /**
+   * Android product flavor to use for the resource resolution, such as "demo".
+   *
+   * @return Flavor used for resource resolution.
+   */
+  String flavor() default DEFAULT_FLAVOR;
```

Comment:
This should probably be `String[]`, right? I can specify multiple flavors using dimensions: http://tools.android.com/tech-docs/new-build-system/user-guide#TOC-Multi-flavor-variants